### ISSUE TITLE
[Docs]: Spinner Migration Doc does not include guidance for migrating spinners that have aria-labels and aria-lives from v8 to v9

### ISSUE THREAD

**Issue by snisonoffMS** [microsoft/fluentui#33835]
### Area

React Components (https://react.fluentui.dev)

### What kind of documentation issue are you reporting?

- [ ] Reporting a typo
- [x] Reporting a documentation bug
- [x] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?

https://react.fluentui.dev/?path=/docs/concepts-migration-from-v8-components-spinner-migration--docs

### Description

The v9 Spinner does not have an aria-label nor aria-live for the spinner and does not have any suggestions for consumers using the v8 with these props on how to migrate, thus the doc is incomplete.

### Validations

- [x] Check that there isn't already an issue that requests the same feature to avoid creating a duplicate.

**Comment by CampbellOwen** [comment]
@snisonoffMS Is there an issue with `aria-label` not being applied to the v9 Spinner?

I tried it in a stackblitz and it appears to work correctly: https://stackblitz.com/edit/epecwzke?file=src%2Fexample.tsx

**Comment by snisonoffMS** [comment]
@CampbellOwen You seem to be confusing this issue with https://github.com/microsoft/fluentui/issues/33841 - feel free to close that one because I talked to Sarah about it and it turned out my expected behavior was wrong.

This bug is for the Migration Docs, the fact that the docs imply that the aria-label is gone, when it is not.

**Comment by snisonoffMS** [comment]
@marcosmoura any updates?

**Comment by snisonoffMS** [comment]
keep open
